cmake/modules/FindADOLC.cmake

```diff
@@ -57,7 +57,7 @@ IF(EXISTS ${ADOLC_SETTINGS_H})
   # Check whether ADOL-C is configured with extra trig functions
   #
   FILE(STRINGS "${ADOLC_SETTINGS_H}" ADOLC_ATRIG_ERF_STRING
-    REGEX "#define ATRIG_ERF"
+    REGEX "^[ \t]*#[ \t]*define ATRIG_ERF"
```

Comment:
While at it, would it make sense to change the single space between `define` and the macro by something like `[ \t]\+` as well?